e2e/components/menu/menu-page.ts

```diff
@@ -0,0 +1,59 @@
+import ElementFinder = protractor.ElementFinder;
+
+export class MenuPage {
+
+  constructor() {
+    browser.get('/menu');
+  }
+
+  menu() { return element(by.css('.md-menu')); }
+
+  trigger() { return element(by.id('trigger')); }
+
+  triggerTwo() { return element(by.id('trigger-two')); }
+
+  body() { return element(by.tagName('body')); }
+
+  firstItem() { return element(by.id('one')); }
+
+  secondItem() { return element(by.id('two')); }
+
+  thirdItem() { return element(by.id('three')); }
```

Comment:
They were being set in the menu-e2e.html. Removed the IDs in favor of this though.